glide.yaml

```diff
@@ -91,6 +92,9 @@ import:
 - package: google.golang.org/appengine/datastore
   version: 2e4a801b39fc199db615bfca7d0b9f8cd9580599
 
+- package: github.com/pborman/getopt
+  version: ec82d864f599c39673eef89f91b93fa5576567a1
```

Comment:
mind using a version instead?

Reply:
Doesn't look like getopt has any semver or tags though.

Reply:
hm does the branch `v2` work? no biggie if not .